.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
- [ ] OPTIONAL
Please endeavor to keep this file's directory in the .gitignore file so as to keep your project structure clean

Reply:
- Noted I will not forget that thank you! 🤝